bayesian/v2/api_v2.py

```diff
@@ -15,29 +15,37 @@
 #
 """Definition of all v2 REST API endpoints of the server module."""
 
```

Comment:
for Path and file name.. refer https://github.com/fabric8-analytics/fabric8-analytics-server/pull/610

Reply:
Comment accepted.